cf-serverd/server_tls.c

```diff
@@ -429,7 +429,7 @@ int ServerTLSSessionEstablish(ServerConnectionState *conn)
         int ret = SSL_accept(ssl);
         if (ret <= 0)
         {
-            TLSLogError(ssl, LOG_LEVEL_ERR,
+            (void)TLSLogError(ssl, LOG_LEVEL_ERR,
```

Comment:
@jimis I disagree and in fact this is just a comment to ensure that error codes are handled unless you are explicitly ignoring those. Some static analyzers will warn about this and this is not a good practice to skip checking return codes. 
In some cases you don't care but IMHO you should write this explicitly. 

This code will be removed anyway so I wont argue more here...